### PR TITLE
Model/add audio converter model

### PR DESCRIPTION
**Pull request by Enrique-C**

### CODE REVIEW

File: src/main/java/com/jalasoft/jfc/model/audio/AudioConverter.java [hunk 1]
```diff
@@ -38,6 +62,75 @@
...
+            }
+
+            zipFile(audioParam);
+
+            FolderRemover.removeFolder(audioParam.getOutputPathFile() + audioParam.getFolderName());
+
+            fileResponse.setName(audioParam.getOutputName());
+            fileResponse.setStatus(MessageResponse.SUCCESS200.getMessageResponse());
+            fileResponse.setDownload(zipPath);
+        } catch (Exception e) {
```
Comment (oscarlopez8) [inline src/main/java/com/jalasoft/jfc/model/audio/AudioConverter.java:96]: "Exception" must be someone of our exceptions

File: src/main/java/com/jalasoft/jfc/model/audio/AudioConverter.java [hunk 2]
```diff
@@ -38,6 +62,75 @@
...
+            throw new ConvertException("Error converting Audio: " + e.getMessage(), this.getClass().getName());
+        }
+        return fileResponse;
+    }
+
+    /**
+     * Generates a command to convert an audio to another audio.
```
Comment (oscarlopez8) [inline src/main/java/com/jalasoft/jfc/model/audio/AudioConverter.java]: Change Generates a command to convert an audio to another audio **to** **Generates a command to convert a file audio to another "format" of file audio**

File: src/main/java/com/jalasoft/jfc/model/audio/AudioFormat.java [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+ * This software is the confidential and proprietary information of Jalasoft.
+ * ("Confidential Information"). You shall not disclose such Confidential
+ * Information and shall use it only in accordance with the terms of the
+ * license agreement you entered into with Jalasoft.
+ */
+
+package com.jalasoft.jfc.model.audio;
+
+public enum AudioFormat {
+    MKV(".wav"),
```
Comment (oscarlopez8) [inline src/main/java/com/jalasoft/jfc/model/audio/AudioFormat.java]: Change MKV to WAV

File: src/main/java/com/jalasoft/jfc/model/audio/AudioFormat.java [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+    VID(".mp3");
```
Comment (oscarlopez8) [inline src/main/java/com/jalasoft/jfc/model/audio/AudioFormat.java]: Change VID to MP3

File: src/main/java/com/jalasoft/jfc/model/audio/AudioFormat.java [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+
+    // Command variable
+    private String format;
+
+    /**
+     * This method is for setting command value.
```
Comment (oscarlopez8) [inline src/main/java/com/jalasoft/jfc/model/audio/AudioFormat.java]: Delete "This method is for"

File: src/main/java/com/jalasoft/jfc/model/audio/AudioFormat.java [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+     * @param format String value
+     */
+    AudioFormat(String format) {
+        this.format = format;
+    }
+
+    /**
+     * This method gets command value.
```
Comment (oscarlopez8) [inline src/main/java/com/jalasoft/jfc/model/audio/AudioFormat.java]: Change This method gets command value. to **Gets command value.**

File: src/main/java/com/jalasoft/jfc/model/audio/AudioParam.java [hunk 7]
```diff
@@ -20,20 +20,39 @@
  */
 public class AudioParam extends Param {
 
+    //Contains format of an audio.
```
Comment (oscarlopez8) [inline src/main/java/com/jalasoft/jfc/model/audio/AudioParam.java]: Add space //Contains 
// Contains

File: src/main/java/com/jalasoft/jfc/model/command/ffmpeg/CommandAudioBitRate.java [hunk 8]
```diff
@@ -0,0 +1,59 @@
...
+    /**
+     * Creates a new CommandAudioBitRate object.
+     * @param commandValue receive a value.
+     */
+    public CommandAudioBitRate(String commandValue) {
+        this.commandValue = commandValue;
+    }
+
+    /**
+     * This builds a command.
```
Comment (oscarlopez8) [inline src/main/java/com/jalasoft/jfc/model/command/ffmpeg/CommandAudioBitRate.java]: Delete "This "

File: src/main/java/com/jalasoft/jfc/model/audio/AudioConverter.java (hunk 1 above)
Comment (Enrique-C) [reply src/main/java/com/jalasoft/jfc/model/audio/AudioConverter.java:96]: This code needs that exception "audioConvertProcess.waitFor();"

File: src/main/java/com/jalasoft/jfc/model/audio/AudioConverter.java (hunk 2 above)
Comment (Enrique-C) [reply src/main/java/com/jalasoft/jfc/model/audio/AudioConverter.java]: That method only generates commands

**Review by oscarlopez8 — APPROVED** [review]

**Pull request merged**